spec/factories/rounds.rb

```diff
@@ -27,5 +37,6 @@
     members_can_propose_buckets true
```

Comment:
similarly, if a round is closed, shouldn't `members_can_propose_buckets` be false?

if this is the case, then can't `members_can_propose_buckets` be a computed value based on whether the round is in 'proposal mode' or not?